volatility3/framework/plugins/windows/handles.py

```diff
@@ -52,6 +52,12 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
             requirements.PluginRequirement(
                 name="pslist", plugin=pslist.PsList, version=(2, 0, 0)
             ),
+            requirements.ListRequirement(
+                name="type",
```

Comment:
Sorry, minor nitpick, since this is a list, we should call it `types` not just `type` so people are aware they can enter multiple values...

Reply:
I agree that that's an improvement, I had originally used `type` because we use `pid` for the other list requirement which accepts multiple values. I do see that we use the singular `pid` for at least all windows plugins.

I pushed a fix and then squashed the change back down to a single commit.